src/Utils/ProtectedRoutes.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Its probably a matter of taste, but I think lowercase folder names are more standard: `utils`